carina-demo/src/main/java/com/qaprosoft/carina/demo/gui/pages/OnlinerComparePage.java

```diff
@@ -9,15 +9,15 @@
 
 public class OnlinerComparePage extends AbstractPage {
 
-    @FindBy(xpath = "//tr[@class='product-table__row product-table__row_header product-table__row_top']" +
-            "/th[@class='product-table__cell']")
+    @FindBy(xpath = "//div[@class='product-table-cell-container']")
```

Comment:
use className

Reply:
also I would propose to move the product with its chars into a separate component